dashboard/streamlit_app.py

```diff
@@ -189,6 +190,33 @@ def get_data_from_db(conn: connect) -> pd.DataFrame:
     return df
 
 
+def generate_word_cloud(summary_texts):
+    combined_text = ' '.join(summary_texts)
+    word_cloud = WordCloud(width=800, height=400, background_color='white',
+                           stopwords=STOPWORDS).generate(combined_text)
+    return word_cloud
+
+
+def get_summary_texts_from_db(conn, case_no):
+    summary_texts = []
+    try:
```

Comment:
nice try and except here to ensure no errors are raised when we can't connect to the database.